src/config/musl.md

```diff
@@ -0,0 +1,99 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms except i686. Additionally, all the packages in our official
+repositories are available with musl-linked binaries in addition to their glibc
+counterparts.
```

Comment:
i686-musl works the same as anything else, there are just no binary packages for it

Reply:
Updated.